src/lib/draw/Toolbox.svelte

```diff
@@ -171,6 +175,16 @@
     {/if}
     <SplitRouteMode bind:this={splitRouteMode} {mode} {changeMode} />
   </div>
+  <div>
```

Comment:
I put this new feature as a drawing/editing tool. That's kind of misleading, because it's just for read-only info. But the controls are mutually exclusive with everything else -- it takes over some keybindings and map controls, so the toolbox is kind of our place to do that for now.

Reply:
I think this is fine for now.